### PR TITLE
[LAND-SEE] - Add story categories data, finish building out stories template 

### PR DESCRIPTION
**Pull request by afilbert**
PR adds story category data, category navigation using `amp-bind` and `QUERY_PARAM()`s, and category paging for `amp-list`. This PR should complete the stories template. Staging URL:

https://landseeampstart.firebaseapp.com/templates/land-see/stories.amp.html

I'm using `:target`, as suggested by @camelburrito, along with query parameters to help handle data and nav state depending on if initial page load or user-initiated interaction.

I'm simply removing the initial page load class on user-initiated navigation, making `:target` ineffective, so `amp-bind` can take care of setting the navigation style on each tapped/clicked button thereafter. 

The template loads all stories by default. We can't anticipate that all incoming requests for `stories.amp.html` will contain a location hash, and `:target` causes page jumping. I'm accordingly omitting the nav underline style for "all" on page load from an incoming URL lacking a location hash and/or query parameter.

Finally, I'm using the stories mock api data to handle `amp-list` paging. You can see that in the `all-stories` and `arts-culture` json page files. Since the template wants to populate the `ul` elem once it gets to any paging data in the items array, I'm letting it load an empty `li` and then simply hiding it to avoid invalid markup. You can see that with this line:

`"land-see-category": [{"post-classes": "hide"}]`

### CODE REVIEW

File: css/templates/land-see/page.css [hunk 1]
```diff
@@ -210,7 +213,7 @@ h4,
...
...3,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3Cpath fill='none' stroke='%23EFEFEF' stroke-miterlimit='10' d='M0,29.708c6.468,0,6.468,4,12.935,4c6.468,0,6.468-4,12.936-4 c6.466,0,6.466,4,12.931,4c6.468,0,6.468-4,12.935-4c6.47,0,6.47,4,12.94,4c6.468,0,6.468-4,12.935-4c6.473,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3Cpath fill='none' stroke='%23EFEFEF' stroke-miterlimit='10' d='M0,34.576c6.468,0,6.468,4,12.935,4c6.468,0,6.468-4,12.936-4 c6.466,0,6.466,4,12.931,4c6.468,0,6.468-4,12.935-4c6.47,0,6.47,4,12.94,4c6.468,0,6.468-4,12.935-4c6.473,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3Cpath fill='none' stroke='%23EFEFEF' stroke-miterlimit='10' d='M0,39.443c6.468,0,6.468,4,12.935,4c6.468,0,6.468-4,12.936-4 c6.466,0,6.466,4,12.931,4c6.468,0,6.468-4,12.935-4c6.47,0,6.47,4,12.94,4c6.468,0,6.468-4,12.935-4c6.473,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3Cpath fill='none' stroke='%23EFEFEF' stroke-miterlimit='10' d='M0,44.312c6.468,0,6.468,4,12.935,4c6.468,0,6.468-4,12.936-4 c6.466,0,6.466,4,12.931,4c6.468,0,6.468-4,12.935-4c6.47,0,6.47,4,12.94,4c6.468,0,6.468-4,12.935-4c6.473,0,6.473,4,12.945,4 c6.471,0,6.471-4,12.943-4'/%3E%3C/svg%3E") repeat-x left top local;
```
Comment (camelburrito) [inline css/templates/land-see/page.css]: can you use svgo and optimize this SVG , i think you will get some savings there

File: css/templates/land-see/page.css [hunk 2]
```diff
@@ -712,3 +730,145 @@ h4,
...
+  letter-spacing: 1.4px;
+  border: none;
+  text-transform: uppercase;
+  text-decoration: underline;
+  background-color: transparent;
+  padding: 0 0 3px 0;
+  cursor: pointer
+}
+
+.land-see-next-page:after,
```
Comment (camelburrito) [inline css/templates/land-see/page.css]: ::after, ::before - change everywhere - https://stackoverflow.com/questions/8980475/after-vs-after

File: css/templates/land-see/page.css [hunk 3]
```diff
@@ -712,3 +730,145 @@ h4,
...
+  z-index: var(--z2);
+}
+
+.land-see-overflow-button {
+  color: var(--color-font-secondary);
+  border-color: var(--color-font-secondary);
+  opacity: 1;
+}
+
+@media (max-width: 30rem) {
```
Comment (camelburrito) [inline css/templates/land-see/page.css]: define this breakpoint in your page-vars.css file

File: templates/land-see/land-see.amp.html [hunk 4]
```diff
@@ -117,18 +117,18 @@
...
-        "animations": [
-          {
-            "selector": ".land-see-subscribe-bg",
-            "keyframes": [
-              { "transform": "translateY(calc(height('.land-see-subscribe') * -0.4))" }
-            ]
-          }
-        ]
-      }
+          "duration": "1s",
```
Comment (camelburrito) [inline templates/land-see/land-see.amp.html]: remove tabs and replace with spaces.

File: templates/land-see/stories.amp.html [hunk 5]
```diff
@@ -43,36 +81,131 @@
...
-        "animations": [
-          {
-            "selector": ".land-see-subscribe-bg",
-            "keyframes": [
-              { "transform": "translateY(calc(height('.land-see-subscribe') * -0.4))" }
-            ]
-          }
-        ]
-      }
+          "duration": "1s",
```
Comment (camelburrito) [inline templates/land-see/stories.amp.html]: replace tabs with spaces

File: css/templates/land-see/page.css (hunk 2 above)
Comment (afilbert) [reply css/templates/land-see/page.css]: What, we're not worried about supporting IE8? :P

File: templates/land-see/land-see.amp.html (hunk 4 above)
Comment (afilbert) [reply templates/land-see/land-see.amp.html]: Hmm... those should be spaces. I double checked by showing invisibles in Atom. They appear to be showing up as spaces, there. I have "soft" tabs enabled, too, which should substitute each tab for 2 spaces.

screen shot 2017-10-18 at 8 46 36 pm

File: css/templates/land-see/page.css (hunk 1 above)
Comment (afilbert) [reply css/templates/land-see/page.css]: Nice catch! That should give us some headroom on the 50 KB CSS limit. I think we may be able to bring this template home without exceeding. 🤞

File: templates/land-see/stories.amp.html (hunk 5 above)
Comment (afilbert) [reply templates/land-see/stories.amp.html]: This should be the same as `land-see.amp.html`, featuring spaces.

File: css/templates/land-see/page.css (hunk 1 above)
Comment (camelburrito) [reply css/templates/land-see/page.css]: I’m also not opposed to loading this as a separate Svg image if this is the sole cause of lots of bytes

File: css/templates/land-see/page.css (hunk 2 above)
Comment (camelburrito) [reply css/templates/land-see/page.css]: No , Amp does not support 8 :) only latest 2 versions of any browser

File: css/templates/land-see/page.css (hunk 1 above)
Comment (afilbert) [reply css/templates/land-see/page.css]: Good to know. I'll keep that in mind.

**Comment by spacedino** [comment]
@camelburrito is this fixed?

**Pull request merged**